res/code_templates/field/varchar/custom/Extension/modules/__module__/Ext/Vardefs/sugarfield___field__.php.twig

```diff
@@ -17,12 +17,12 @@ $dictionary["{{ moduleBean }}"]["fields"]["{{ field }}"] = array(
     'comment' => '',
     'help' => '',
     // Sugar search features
-    'unified_search' => true,
-    'full_text_search' => array(
-        'enabled' => true,
-        'searchable' => true,
-        'boost' => 1.00
-    ),
+    //'unified_search' => false,
```

Comment:
I commented the lines as I thought if we need to enable the search feature we can uncomment the lines.